EventFiltering/PWGMM/multFilter.cxx

```diff
@@ -83,6 +87,26 @@ struct multFilter {
   Filter trackFilter = (nabs(aod::track::eta) < cfgTrackEtaCut) && (aod::track::isGlobalTrack == (uint8_t) true) && (aod::track::pt > cfgTrackLowPtCut);
 
   using TrackCandidates = soa::Filtered<soa::Join<aod::Tracks, aod::TrackSelection>>;
+  float computeDeltaPhi(float phia, float phib,
+                        float rangeMin = -M_PI / 2.0, float rangeMax = 3.0 * M_PI / 2.0)
+  {
+    float dphi = -999;
+    if (phia < 0)
```

Comment:
Please note that ALICE coding convention explicitly requires braces in `if () {} else {}` etc. Even if there would be a single line inside the braces.

Reply:
Hello @aalkin, thanks for the suggestions. Please check the updated version.